backend/functions/lib/adapters/gateways/okta/okta-gateway.ts

```diff
@@ -32,25 +32,31 @@ async function verifyToken(token: string): Promise<CamsJwt> {
     throw new ServerConfigError(MODULE_NAME, { message: 'Audience not provided.' });
   }
   try {
-    return await verifyAccessToken(issuer, token, audience);
+    const maybeCamsJwt = await verifyAccessToken(issuer, token, audience);
+    if (maybeCamsJwt.claims.groups) {
+      return maybeCamsJwt as CamsJwt;
+    }
+    throw new Error('Access token claims missing groups.');
```

Comment:
Maybe reverse these 2?  Fail with an error if (!maybeCamsJwt.claims.groups) ?, and otherwise return it?

Reply:
I do prefer the returning early on errors. Terminate with the golden path.